pipeline/utils.py

```diff
@@ -5,22 +5,51 @@
 
 from git import Repo, GitCommandError
 
-source_url = "https://github.com/openMetadataInitiative/openMINDS.git"
-
 
 def clone_sources():
-    if os.path.exists("sources"):
-        shutil.rmtree("sources")
-    Repo.clone_from(source_url, to_path="sources", depth=1)
-
-
-class SchemaLoader(object):
+    if os.path.exists("_sources"):
```

Comment:
I would rename here "_sources_schemas" for the schemas and then use "_sources_instances" for the instances. I think it should not cause too many refactoring for the schemas.